tests/pr_test.go

```diff
@@ -110,9 +110,20 @@ func setupOptions(t *testing.T, prefix string, dir string, terraformVars map[str
 	os.Setenv("TF_VAR_sls_license_id", *slsLicenseId)
 
 	// Deploy Pre-requisite resources
-
 	realTerraformDir := "./resources"
 	tempTerraformDir, _ := files.CopyTerraformFolderToTemp(realTerraformDir, options.Prefix)
+	// Need to replace the symlinked file with the actual file since symlink will break when tests moves files to temp directory
+	override_location := tempTerraformDir + "/override.json"
+	e := os.Remove(override_location)
```

Comment:
I believe `os.Remove()` will return an error if the file does not exist, if that happens that is not really an error for you. If you want to ignore a "file does not exist" here, you can pass the error object to the `IsNotExist()` https://pkg.go.dev/os#IsNotExist function to see if that was the error and then ignore.